### PR TITLE
Something stupid again

### PR DESCRIPTION
**Pull request by 9600bauds**
After a brief discussion with Intigracy we agreed that this was something really stupid and as such I had no choice but to add it
- Catbeasts can now speak Mouse

While I was at it, I found the perfect excuse to pretend like there is an actual logical change here
- Skelevox can now also speak Clatter
- AIs can now understand Golem

And something like I can pretend I'm contributing to the code
- Everything now uses language defines

And finally something fun
- When speaking Mouse, Catbeasts still have their wordfilters applied

### CODE REVIEW

File: code/modules/mob/language.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@
 	syllables = list("ss","ss","ss","ss","skak","seeki","resh","las","esi","kor","sh")
 
 /datum/language/tajaran
-	name = "Siik'tajr"
+	name = LANGUAGE_SIIK_TAJR
```
Comment (Intigracy) [inline code/modules/mob/language.dm]: just call it language_catbeast please

File: code/modules/mob/living/carbon/species.dm [hunk 2]
```diff
@@ -469,7 +469,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Muton"
 	icobase = 'icons/mob/human_races/r_muton.dmi'
 	deform = 'icons/mob/human_races/r_def_muton.dmi'
-	language = "Muton"
+	known_languages = ("Muton")
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/species.dm]: ?

File: code/modules/mob/living/carbon/species.dm [hunk 3]
```diff
@@ -732,7 +732,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Golem"
 	icobase = 'icons/mob/human_races/r_golem.dmi'
 	deform = 'icons/mob/human_races/r_def_golem.dmi'
-	language = "Golem"
+	known_languages = ("Golem")
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/species.dm]: ?

**Comment by clusterfack** [comment]
:full_moon_with_face:

File: code/modules/mob/living/carbon/species.dm (hunk 2 above)
Comment (9600bauds) [inline code/modules/mob/living/carbon/species.dm]: ahahaha this language doesn't even exist, classic goofball!

File: code/modules/mob/living/carbon/monkey/diona.dm [hunk 4]
```diff
@@ -127,7 +127,7 @@
 /mob/living/carbon/monkey/diona/say_understands(var/mob/other,var/datum/language/speaking = null)
 	if(other) other = other.GetSource()
 	if (istype(other, /mob/living/carbon/human))
-		if(speaking && speaking.name == "Galactic Common")
+		if(speaking && speaking.name == LANGUAGE_GALACTIC_COMMON)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/monkey/diona.dm:130]: What's the difference between Galactic Common and Sol Common?

File: code/modules/mob/living/carbon/species.dm [hunk 5]
```diff
@@ -252,7 +252,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Manifested"
 	icobase = 'icons/mob/human_races/r_manifested.dmi'
 	deform = 'icons/mob/human_races/r_def_manifested.dmi'
-	language = "Sol Common"
+	known_languages = LANGUAGE_HUMAN
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: Should be a list.

File: code/modules/mob/living/carbon/species.dm [hunk 6]
```diff
@@ -509,7 +509,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Skrell"
 	icobase = 'icons/mob/human_races/r_skrell.dmi'
 	deform = 'icons/mob/human_races/r_def_skrell.dmi'
-	language = "Skrellian"
+	known_languages = (LANGUAGE_SKRELLIAN)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: Should be a list

File: code/modules/mob/living/carbon/species.dm [hunk 7]
```diff
@@ -520,7 +520,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Vox"
 	icobase = 'icons/mob/human_races/vox/r_vox.dmi'
 	deform = 'icons/mob/human_races/vox/r_def_vox.dmi'
-	language = "Vox-pidgin"
+	known_languages = (LANGUAGE_VOX)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: etc

File: code/modules/mob/living/carbon/species.dm [hunk 8]
```diff
@@ -702,7 +702,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Diona"
 	icobase = 'icons/mob/human_races/r_plant.dmi'
 	deform = 'icons/mob/human_races/r_def_plant.dmi'
-	language = "Rootspeak"
+	known_languages = (LANGUAGE_ROOTSPEAK)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: etc

File: code/modules/mob/living/carbon/species.dm [hunk 9]
```diff
@@ -732,7 +732,7 @@ var/global/list/whitelisted_species = list("Human")
 	name = "Golem"
 	icobase = 'icons/mob/human_races/r_golem.dmi'
 	deform = 'icons/mob/human_races/r_def_golem.dmi'
-	language = "Golem"
+	known_languages = (LANGUAGE_GOLEM)
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: etc

File: code/modules/mob/living/carbon/monkey/diona.dm (hunk 4 above)
Comment (ghost) [inline code/modules/mob/living/carbon/monkey/diona.dm:130]: Sol common is human-only, galactic common is what everyone speaks

**Comment by ghost** [comment]
Do catbeasts still scream shit like PLEASE KILL ME? Always thought that was stupid as hell. Anyway, this is good, time to catbeast myself and lure in silly mice.

File: code/modules/mob/living/carbon/species.dm (hunk 5 above)
Comment (9600bauds) [inline code/modules/mob/living/carbon/species.dm]: Fugg, why did it compile

Comment (ComicIronic) [inline code/modules/mob/living/carbon/species.dm]: Because Byond is not typesafe.

**Comment by gbasood** [comment]
Code looks sane :ok_hand: :100: percent reviewed

**Pull request merged**